### ISSUE TITLE
When 'Creating USB Installer' I get this error /usr/local/lib: Permission denied

### ISSUE THREAD

**Issue by ZinkDifferent** [BenSova/Patched-Sur#750]
### Discussed in https://github.com/BenSova/Patched-Sur/discussions/749

<div type='discussions-op-text'>

<sup>Originally posted by **ZinkDifferent** August 31, 2021</sup>
It happily goes through all the steps, but after reaching 100% at 'Creating USB Installer', it gives me this error, and no means of getting beyond it. This is on a MacBook Pro Retina, mid-2012 running Catalina 10.15.7

Is this SIP (ie csrutil) messing with the installer, or any other ideas?
<img width="712" alt="Screen Shot 2021-08-31 at 12 52 05 PM" src="https://user-images.githubusercontent.com/5345100/131567793-cbce4430-3e14-42dc-a9cf-90b86566001b.png">
</div>

**Comment by ZinkDifferent** [comment]
I was finally able to get past this:
- enabled root user
- logged in as root user

The first time I ran Sur Patcher from the applications folder, it errored out with a 'file not found' error during the patching phase, and I noted that Sur Patcher has disappeared from the Applications folder.

Then I mounted the DMG again, and and ran it from the DMG - it went through just fine, asked to enable root permissions, and created the USB Installer. 

I'm typing this from my 2012 MacBook Pro Retina running Big Sur. Everything (so far) seems to work just fine.

This really should be more clearly stated in the instructions inside the installer of the Sur Patcher. The need to run this from the actual root account (ie being logged in as root, after enabling root), should be made clear.

Thanks for creating this installer.
